app/models/container_project.rb

```diff
@@ -26,6 +28,7 @@ class ContainerProject < ApplicationRecord
   virtual_column :routes_count,      :type => :integer
   virtual_column :replicators_count, :type => :integer
   virtual_column :containers_count,  :type => :integer
+  virtual_column :images_count,      :type => :integer
 
   def groups_count
```

Comment:
not sure if you want to do it but...
all of these could be converted into `virtual_total` columns